test/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Do you need a separate config for tests?

Reply:
I'm sure this used to be necessary for jest or something so it could run tests - doesn't seem to be needed now, so will remove